cpp/basix/e-crouzeix-raviart.cpp

```diff
@@ -63,6 +64,13 @@ FiniteElement basix::create_cr(cell::type celltype, int degree)
   }
 
   M[tdim - 1].resize(facet_topology.size(), xt::ones<double>({1, 1, 1}));
+
+  if (discontinuous)
```

Comment:
Could this be moved higher up?

There are some more cases below where I think the exception can be raised earlier - it would be good to place it as early as possible in functions.

Reply:
This one can't as it needs x, M, and entity_transformations to be made before it so they can then be updated to make the discontinuous element. I've moved up the exceptions that could be moved though